### PR TITLE
Add $font-host variable to fonts file

### PR DESCRIPTION
**Pull request by RoyEJohnson**
Issue #39
Allows configuration of font file loader

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "pattern-library",
-  "version": "1.1.16",
+  "version": "1.1.17",
```
Comment (BryanEliDimas) [inline package.json:3]: Please also update the gemspec file (ruby gem version) accordingly.

File: core/pattern-library/elements.scss [hunk 2]
```diff
@@ -1,4 +1,5 @@
 @import 'headers';
+// $font-host: 'https://openstax.org';
```
Comment (BryanEliDimas) [inline core/pattern-library/elements.scss:2]: Is it intentionally commented out? If so, please explain why or what it means/does?

File: core/pattern-library/fonts.scss [hunk 3]
```diff
@@ -1,53 +1,54 @@
 @import url('https://fast.fonts.net/t/1.css?apiType=css&projectid=437b6557-ce99-4f35-97ff-64a93247731f');
+$font-host: '' !default;
```
Comment (BryanEliDimas) [inline core/pattern-library/fonts.scss:2]: Please do set an actual default value? 👀 Or, I don't know what that does ^

**Comment by BryanEliDimas** [comment]
cc: @edwoodward Relevant for the font problem you're trying to fix in Accounts.

File: core/pattern-library/elements.scss (hunk 2 above)
Comment (RoyEJohnson) [reply core/pattern-library/elements.scss:2]: It's there as an example but isn't needed for the online demo.

File: core/pattern-library/fonts.scss (hunk 3 above)
Comment (RoyEJohnson) [reply core/pattern-library/fonts.scss:2]: The default value of empty is "current host", which I think is the right default.

**Pull request merged**